integration/bin/test/get-circle-string-artifact-url.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import child_process from 'child_process';
+import path from 'path';
+
+const scriptPath = path.join( '.', 'bin', 'get-circle-string-artifact-url' );
+
+describe( 'get-circle-string-artifact-url', () => {
+	it( 'We can fetch translation strings from CircleCi artifacts', () => {
+		const url = child_process.execSync( `node ${ scriptPath }` ).toString().trim();
```

Comment:
doesn't `node` normalize paths on `/` so that we could inline this path?

Reply:
> The path.join() method joins all given path segments together using the platform specific separator as a delimiter, then normalizes the resulting path.

https://nodejs.org/api/path.html#path_path_join_paths

Reply:
someone told me that `node` itself doesn't require this as it does it automatically. I can't find any docs specifying one way or the other.